src/pages/[[...slug]].tsx

```diff
@@ -1,22 +1,46 @@
 import { FC } from 'react';
+import Head from 'next/head';
 import { GetStaticPaths, GetStaticProps } from 'next';
 import { sourcebitDataClient } from 'sourcebit-target-next';
 import { hotContentReload } from 'sourcebit-target-next/hot-content-reload';
-
-import type { PageProps } from '../components/layouts';
-import { BaseLayout } from '../components/layouts/BaseLayout';
-import { PageLayout, Props as PageLayoutProps } from '../components/layouts/PageLayout';
+import { toObjectId, toFieldPath } from '@stackbit/annotations';
+import { Header } from '../components/sections/Header';
+import { Footer } from '../components/sections/Footer';
+import { DynamicComponent } from '../components/DynamicComponent';
 import { findPageLayouts, toPageProps, urlPathForDocument } from '../utils/static-resolver-utils';
-import { mapProps as mapPageLayoutProps } from '../components/layouts/PageLayout/mapProps';
+import * as types from 'types';
+
+import MuiBox from '@mui/material/Box';
+import MuiContainer from '@mui/material/Container';
 
-export type Props = PageProps<PageLayoutProps>;
+export type Props = {
+    site: types.Config & { env?: Record<string, string> };
```

Comment:
Since all pages will probably want the env, this "union" type should already be defined in types and not here.
Actually we could define an interface such as "BasePageProps" that already has site & env inside, and then for the specific page's props we just add "page"